src/components/entry/EntryNav.js

```diff
@@ -1,7 +1,70 @@
-const EntryNav = () => (
-  <>
-    <p>EntryNav</p>
-  </>
-)
+import { ArrowBack } from '@styled-icons/boxicons-regular'
+import { Pencil } from '@styled-icons/boxicons-solid'
+import { MapMarkerAlt as Map } from '@styled-icons/fa-solid'
+import styled from 'styled-components'
+
+import { theme } from '../ui/GlobalStyle'
+import IconButton from '../ui/IconButton'
+
+const EntryNavContainer = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
You can omit the `flex-direction: row` in all the containers because it's the default.